nectar/downloaders/threaded.py

```diff
@@ -56,6 +45,11 @@ def __str__(self):
 class SkipLocation(Exception):
     pass
 
+
+class RetryError(Exception):
```

Comment:
seems like everything that lives in exceptions.py is not used anywhere. At least i don't see that it is imported somewhere:
```
git grep exceptions
LICENSE.txt:make exceptions for this.  Our decision will be guided by the two goals
nectar/exceptions.py:# specific derived exceptions
nectar/exceptions.py:# specific derived exceptions
nectar/exceptions.py:# specific derived exceptions
nectar/exceptions.py:# specific derived exceptions
```
For now I will leave this as it is.